easybuild/easyconfigs/o/OpenMPI/OpenMPI-4.1.4-GCC-11.3.0.eb

```
@@ -36,7 +36,15 @@ dependencies = [
     ('UCC', '1.0.0'),
 ]
 
-preconfigopts = './autogen.pl --force && '
+preconfigopts = ' && '.join([
+    'cd config',
+    'autom4te --language=m4sh opal_get_version.m4sh -o opal_get_version.sh; cd ..',
```

this is just cosmetic, but having just these two lines combined with a `;` looks a little inconsistent.

---

My idea was that a failing `autom4ate` call should still result in going back, but that doesn't actually matter, so:

```suggestion
    'autom4te --language=m4sh opal_get_version.m4sh -o opal_get_version.sh',
    'cd ..',
```